### PR TITLE
Remove unneeded cancel all task call

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description
This is already called in the base class's finalize_run() method.
So no need to call it again in stop_controller() method

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/app_common/workflows/scatter_and_gather.py [hunk 1]
```diff
@@ -291,9 +291,8 @@ def control_flow(self, abort_signal: Signal, fl_ctx: FLContext) -> None:
             self.log_exception(fl_ctx, error_msg)
             self.system_panic(error_msg, fl_ctx)
 
-    def stop_controller(self, fl_ctx: FLContext) -> None:
+    def stop_controller(self, fl_ctx: FLContext):
```
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply nvflare/app_common/workflows/scatter_and_gather.py:294]: the `cancel_all_tasks` is already called in base class's `finalize`
the `stop_controller` will be called in base class's `finalize` afterwards, 
so no need to call it twice.

File: nvflare/app_common/workflows/broadcast_and_process.py [hunk 2]
```diff
@@ -120,8 +120,8 @@ def _process_client_response(self, client_task: ClientTask, fl_ctx: FLContext) -
                 reason=f"ResponseProcessor {type(self.processor)} failed to check client {client.name}", fl_ctx=fl_ctx
             )
 
-    def stop_controller(self, fl_ctx: FLContext) -> None:
-        self.cancel_all_tasks()
+    def stop_controller(self, fl_ctx: FLContext):
```
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply nvflare/app_common/workflows/broadcast_and_process.py:123]: "stop_controller()" is only used in the parent class's method "finalize_run()".

And the "finalize_run()" method will always call "cancel_all_tasks()" first.

So we don't need to duplicate this inside child's stop_controller method

**Review by yhwen — APPROVED** [review]

**Pull request merged**